lib/server/routers/index.js

```diff
@@ -29,6 +29,7 @@ const { statisticsRouter } = require('./statistics.router.js');
 const statusRoute = require('./status.router');
 const subsystemsRoute = require('./subsystems.router');
 const tagsRoute = require('./tags.router');
+const { lhcPeriodsRouter } = require('./lhcPeriods.router.js');
```

Comment:
Please rename all the files and exports when it applies

Reply:
applied